ui/src/components/helpers/customAxios.js

```diff
@@ -4,7 +4,7 @@ export let customAxios;
 
 export let updateAxiosClient = (token) => {
   customAxios = axios.create({
-    headers: { Authorization: "Bearer " + token },
+    headers: { Authorisation: "Bearer " + token },
```

Comment:
should be "Authorization" right?

Reply:
nope, it's actually authorisation. Authorization causes issues